chia/plotters/madmax.py

```diff
@@ -160,7 +160,9 @@ def plot_madmax(args, chia_root_path: Path, plotters_root_path: Path):
 
         # madMAx has a ulimit -n requirement > 296:
         # "Cannot open at least 296 files, please raise maximum open file limit in OS."
-        resource.setrlimit(resource.RLIMIT_NOFILE, (512, 512))
+        soft_limit, hard_limit = resource.getrlimit(resource.RLIMIT_NOFILE)
+        # Set soft limit to max (hard limit)
+        resource.setrlimit(resource.RLIMIT_NOFILE, (hard_limit, hard_limit))
```

Comment:
I'm not really sure that jumping to a limit of 1M (maybe) on my system is really an appropriate response?

Reply:
i'm not sure what madmax needs wrt/file handle, especially at different bucket sizes so hard to say what number to use aside from the hard limit. i'm not worried about it going off the rails wrt/file handles. i think that is all the soft limit is trying to ensure.

Reply:
Seems like something to fix upstream?

Reply:
Should be fixed now.